src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--start-day-sun calendar--month-length-31">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
Remove it everywhere
```suggestion
        <div class="calendar__day"></div>
```